packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -309,10 +320,12 @@ class _InkResponseState<T extends InkResponse> extends State<T> {
       widget.onDoubleTap();
   }
 
-  void _handleLongPress() {
+  void _handleLongPress(BuildContext context) {
     _currentSplash?.confirm();
     _currentSplash = null;
     if (widget.onLongPress != null)
+      if (widget.enableFeedback)
+        Feedback.forLongPress(context);
       widget.onLongPress();
```

Comment:
In that case maybe we should just `assert(widget.onLongPress != null)` and remove the test?